Source/RunActivity/Viewer3D/Processes/GameStateViewer3D.cs

```diff
@@ -30,10 +30,11 @@ public class GameStateViewer3D : GameState
         bool FirstFrame = true;
         int ProfileFrames = 0;
 
-        public GameStateViewer3D(Viewer viewer)
+        public GameStateViewer3D(Viewer viewer, bool paused)
         {
             Viewer = viewer;
-            Viewer.Simulator.Paused = false;
+            Viewer.Simulator.Paused = paused;
```

Comment:
As far as I can tell, all the places that call this do so with `paused = true`, so I'd prefer just coding that in here instead of passing a `bool` in.

Reply:
This change was made before it was agreed to apply to all start/resume moments. So at first it was only appled for timetable mode and therefor required it to be passed in.